package.json

```
@@ -44,8 +44,8 @@
     "@hapipal/toys": "3.x.x",
     "bin-v8-flags-filter": ">=1.2.0 <2",
     "glob": "7.x.x",
-    "marked": "1.x.x",
-    "marked-terminal": "4.x.x",
+    "marked": "4.0.16",
+    "marked-terminal": "5.x.x",
```

The only v5 marked-terminal versions that have support for CJS usage is v5.1.0+, so I think we'll want something like this:
```suggestion
    "marked-terminal": "^5.1.0",
```

---

Good catch - I'll push up a fix shortly.